package.json

```diff
@@ -2,7 +2,7 @@
   "name": "keysim",
   "version": "1.3.0",
   "description": "Keyboard simulator for JavaScript.",
-  "main": "dist/keysim.js",
+  "main": "lib/keysim.js",
```

Comment:
`lib/keysim.js` is ES6, which node cannot run. Even node v5.x cannot handle `import` and `export` statements, so this needs to continue pointing to a built file.